### PR TITLE
E1.33 cherry pick vii

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

File: libs/acn/LLRPProbeReplyPDUTest.cpp [hunk 1]
```diff
@@ -176,6 +176,14 @@ void LLRPProbeReplyPDUTest::testPrepend() {
     0xff
   };
   OLA_ASSERT_DATA_EQUALS(expected_data, sizeof(expected_data), buffer, length);
+
+  // test null stack
+  LLRPProbeReplyPDU::PrependPDU(
```
Comment (kripton) [inline libs/acn/LLRPProbeReplyPDUTest.cpp:181]: does this test actually test something? shouldn't it check the return value?

File: libs/acn/LLRPProbeRequestPDUTest.cpp [hunk 2]
```diff
@@ -191,6 +191,15 @@ void LLRPProbeRequestPDUTest::testPrepend() {
     0x56, 0x78, 0, 0, 0, 2,
   };
   OLA_ASSERT_DATA_EQUALS(expected_data, sizeof(expected_data), buffer, length);
+
+  // test null stack
+  LLRPProbeRequestPDU::PrependPDU(NULL,
```
Comment (kripton) [inline libs/acn/LLRPProbeRequestPDUTest.cpp:196]: does this test actually test something? shouldn't it check the return value?

File: tools/e133/ManagementEndpoint.h [hunk 3]
```diff
@@ -30,7 +30,7 @@ using ola::rdm::RDMResponse;
 using ola::rdm::RDMCallback;
 
 /**
- * The ManagementEndpoint handles RDMCommands directed at this E1.33 device. It
+ * The ManagementEndpoint handles RDMCommands directed at this E1.33 Component. It
```
Comment (kripton) [inline tools/e133/ManagementEndpoint.h:33]: capital C is on purpose?

**Review by kripton — APPROVED** [review]
see inline comments

File: libs/acn/LLRPProbeReplyPDUTest.cpp (hunk 1 above)
Comment (peternewman) [reply libs/acn/LLRPProbeReplyPDUTest.cpp:181]: There is no return value, it's a void. It only modifies stack, which we've passed in as null anyway, so no change there. I guess we could prove the stack was still null, but I'm not sure it's very beneficial.

Without the null fix in the code though, the test itself segfaults too, so we'll know its broken...

File: libs/acn/LLRPProbeRequestPDUTest.cpp (hunk 2 above)
Comment (peternewman) [reply libs/acn/LLRPProbeRequestPDUTest.cpp:196]: As above

File: tools/e133/ManagementEndpoint.h (hunk 3 above)
Comment (peternewman) [reply tools/e133/ManagementEndpoint.h:33]: Yeah, I think from the spec that both Controllers and Devices now have management endpoints, I believe Component (or RPT Component) is the overarching term for Controller and/or Device.

Comment (peternewman) [reply tools/e133/ManagementEndpoint.h:33]: "thing" would also be acceptable! ;)

Comment (kripton) [reply tools/e133/ManagementEndpoint.h:33]: I rather meant that the previous "device" was lower-case and the new "Component" is upper-case. No issue with the wording itself

Comment (peternewman) [reply tools/e133/ManagementEndpoint.h:33]: I think that was device in the generic sense, whereas I was capitalising the official term from the standard to match the standard.

**Comment by peternewman** [comment]
Merging given @kripton was happy and I ought to try and get the rest merged in...

**Pull request merged**